apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -79,21 +89,32 @@
     max-width: 500px;
 }
 
-.c-contentPage-header-description {
-    @include f.font-size('body-l');
-    margin-top: f.spacing(e);
-    max-width: 50ch;
-
-    @include f.media('>=wide') {
-        @include f.font-size('heading-s');
-    }
-}
 
 .c-contentPage-content {
     flex: 1;
     background-color: f.$color-background-default;
     border-top: 1px solid f.$color-border-default;
 
+    & a {
+        color: f.$color-content-link-distinct;
+
+        &:visited {
+            color: f.$color-content-link-visited;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: none;
+        }
+    }
+
+    & p:has(+ .c-tokensTable) {
+        // For a tokens table directly following a description add margin.
```

Comment:
The wording isn't super clear on this; the CSS reads like it adds margin to the description but the comment makes it sound like it's to the table, so I'm not sure this comment is that useful (the line below is fine because it gives extra context)

Reply:
Just noticed that the comment was already there, you've just reformatted it.

Reply:
Yeh, a lot of these changes have come from switching to `stylelint-config-pie`. I won't update the comment, as I'm not sure as to the exact styling myself, so wouldn't be confident I was making the comment better or worse!